web/src/components/storage/VolumeForm.jsx

```diff
@@ -719,17 +719,26 @@ export default function VolumeForm({ id, volume: currentVolume, templates = [],
 
   const { fsType, snapshots } = state.formData;
 
+  const ShowMountPointSelector = () => (
+    <MountPointFormSelect
+        id="mountPoint"
+        value={state.formData.mountPoint}
+        onChange={changeVolume}
+        volumes={currentVolume ? [currentVolume] : templates}
+        isDisabled={currentVolume !== undefined}
+    />
+  );
+
+  const ShowMountPoint = () => <p>{state.formData.mountPoint}</p>;
+
   return (
     <Form id={id} onSubmit={submitForm}>
       <FormGroup isRequired label={_("Mount point")} fieldId="mountPoint">
-        {currentVolume !== undefined} && <p> {state.formData.mountPoint} </p>
-        {/* <MountPointFormSelect
-          id="mountPoint"
-          value={state.formData.mountPoint}
-          onChange={changeVolume}
-          volumes={currentVolume ? [currentVolume] : templates}
-          isDisabled={currentVolume !== undefined}
-        /> */}
+        <If
+        condition={currentVolume !== undefined}
+        then={<ShowMountPoint />}
+        else={<ShowMountPointSelector />}
+        />
```

Comment:
I suspect linter (ESLint in this case) is going to complain here, because indentation. You can check by yourself by running `npm run eslint`.

Reply:
I did check that before and it didn't show any error.
![Uploading Screenshot 2024-01-09 180201.png…]()

Reply:
Sadly the screenshot was not attached. Maybe I'm wrong, let me check it by myself.

Reply:
sorry sir, I think there was some issue
![Screenshot 2024-01-09 180201](https://github.com/openSUSE/agama/assets/125825782/75d2f4c5-ceb9-4bc7-9d3a-1de1cbdae9fe)

Reply:
Great! You were right. My apologies :innocent: :pray: